File: purchase_interface/src/main/java/com/hanjixin/core/service/SalesService.java
package com.hanjixin.core.service;

import entity.ChartResult;
import entity.MorrisData;

import java.util.List;

public interface SalesService {
    List<MorrisData> salesPre(String start, String end,String username);

    ChartResult salesChart(String start, String end);
}

File: purchase_web_shop/src/main/webapp/js/service/salesService.js
//服务层
app.service('salesService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.sales_pre=function(data){
		return $http.post('../sales/salespre.do?startdata='+data.start+"&enddata="+data.end);
	}
});
